src/status_im/utils/ethereum/tokens.cljs

```diff
@@ -492,15 +492,18 @@
                     :symbol   :MOKSHA
                     :decimals 18
                     :address  "0x6ba7dc8dd10880ab83041e60c4ede52bb607864b"}
-                   {:symbol  :KDO
-                    :nft?    true
-                    :name    "KudosToken"
-                    :address "0x93bB0AFbd0627Bbd3a6C72Bc318341D3A22e254a"}])
+                   {:symbol   :KDO
+                    :nft?     true
+                    :name     "KudosToken"
+                    :address  "0x93bb0afbd0627bbd3a6c72bc318341d3a22e254a"}])
 
    :custom  []})
 
-(defn tokens-for [chain]
-  (get all chain))
+(defn tokens-for
+  "makes sure all addresses are lower-case
+   TODO: token list should be speced and not accept non-lower-cased addresses"
```

Comment:
Some of them were in mixed case intentionally, so that we'd be able to detect if everything still works with [checksummed](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md) addresses. I agree we should do that consistently though and only store normalized version.

Reply:
Oooooh, this makes sense. Hmmmm it would really be nice to know how and where checksummed contract addresses are supposed to flow through the System.